nipype/interfaces/traits_extension.py

```diff
@@ -33,7 +33,7 @@
 from traits.api import BaseUnicode
 from traits.api import Unicode
 
-DictStrStr = traits.Dict(str, (bytes, str))
+DictStrStr = traits.Dict((bytes,str,unicode), (bytes,str,unicode))
```

Comment:
unicode does not exist in python 3 anymore.

you can do

```
import sys

DictStrStr = traits.Dict(str, (bytes,str))
if sys.version_info[0] < 3:
    DictStrStr = traits.Dict((bytes,str,unicode), (bytes,str,unicode))
```

However, ```DictStrStr = traits.Dict((bytes, str), (bytes,str))``` does not solve your problem as well?